src/main/java/edu/rosehulman/covidtracer/model/ContactTracer.java

```diff
@@ -39,12 +42,23 @@ public void setID(Long iD) {
 		ID = iD;
 	}
 
-	public SystemUser getUser() {
-		return user;
+	public Person getUser() {
+		return person;
+	}
+
+	public void setUser(Person person) {
+		this.person = person;
 	}
 
-	public void setUser(SystemUser user) {
-		this.user = user;
+	public double findCaseRatio() {
```

Comment:
Case ratio should be something calculated: 
eg. current number of cases / max cases allowed

Reply:
Wasn't aware this was a thing until now. We  should probably take care of this in another branch when we add the ability to assign a case to a tracer